coding/base64.cpp

```diff
@@ -8,7 +8,7 @@
 #pragma clang diagnostic ignored "-Wunused-local-typedef"
 #endif
 
-#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/trim.hpp>
```

Comment:
Can strings::Trim be used here? Is it header-only?

Reply:
Probably, but I don't see a reason to do it.

Reply:
Because it's been sitting here for 7 years already.
Note that this change is not strictly needed. My code highlighting was messed up until I touched that header but since the change reduces the amount of included things I decided to keep it.